{"success": false, "error": "Traceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 544, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 826, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 86, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 637, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 117, in __init__\n    self._compile(**kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 548, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 154, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 276, in _get_targets_json\n    return _run_solc(\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 548, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Multiple SPDX license identifiers found in source file. Use \"AND\" or \"OR\" to combine multiple licenses. Please see https://spdx.org for more information.\n--> /Users/giacomoibba/Desktop/mainnet/2a/2A2bEbb9582C1b101C2CF7406ddE19A77BBee85e_WonderFi.sol\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/utils/Context.sol\" not found: File not found.\n --> /Users/giacomoibba/Desktop/mainnet/2a/2A2bEbb9582C1b101C2CF7406ddE19A77BBee85e_WonderFi.sol:5:1:\n  |\n5 | import \"../utils/Context.sol\";\n  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@uniswap/v3-core/contracts/interfaces/callback/IUniswapV3SwapCallback.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2A2bEbb9582C1b101C2CF7406ddE19A77BBee85e_WonderFi.sol:199:1:\n    |\n199 | import '@uniswap/v3-core/contracts/interfaces/callback/IUniswapV3SwapCallback.sol';\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@openzeppelin/contracts/token/ERC20/IERC20.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2A2bEbb9582C1b101C2CF7406ddE19A77BBee85e_WonderFi.sol:266:1:\n    |\n266 | import '@openzeppelin/contracts/token/ERC20/IERC20.sol';\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@uniswap/v3-periphery/contracts/libraries/TransferHelper.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2A2bEbb9582C1b101C2CF7406ddE19A77BBee85e_WonderFi.sol:450:1:\n    |\n450 | import '@uniswap/v3-periphery/contracts/libraries/TransferHelper.sol';\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@openzeppelin/contracts/access/Ownable.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2A2bEbb9582C1b101C2CF7406ddE19A77BBee85e_WonderFi.sol:451:1:\n    |\n451 | import \"@openzeppelin/contracts/access/Ownable.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@openzeppelin/contracts/utils/Context.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2A2bEbb9582C1b101C2CF7406ddE19A77BBee85e_WonderFi.sol:452:1:\n    |\n452 | import \"@openzeppelin/contracts/utils/Context.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/2a/VerifierSignature.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2A2bEbb9582C1b101C2CF7406ddE19A77BBee85e_WonderFi.sol:453:1:\n    |\n453 | import './VerifierSignature.sol';\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@uniswap/v3-periphery/contracts/libraries/TransferHelper.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2A2bEbb9582C1b101C2CF7406ddE19A77BBee85e_WonderFi.sol:519:1:\n    |\n519 | import '@uniswap/v3-periphery/contracts/libraries/TransferHelper.sol';\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@uniswap/v3-periphery/contracts/interfaces/ISwapRouter.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2A2bEbb9582C1b101C2CF7406ddE19A77BBee85e_WonderFi.sol:520:1:\n    |\n520 | import '@uniswap/v3-periphery/contracts/interfaces/ISwapRouter.sol';\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/2a/Withdraw.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2A2bEbb9582C1b101C2CF7406ddE19A77BBee85e_WonderFi.sol:521:1:\n    |\n521 | import './Withdraw.sol';\n    | ^^^^^^^^^^^^^^^^^^^^^^^^\n\n\n", "results": {}}
